includes/definitions/discovery/siteboss.yaml

```diff
@@ -0,0 +1,15 @@
+mib: SITEBOSS-530-STD-MIB
+modules:
+    sensors:
+        temperature:
+            data:
+                -
+                    oid: esPointTable
+                    value: esPointValueInt
+                    num_oid: .1.3.6.1.4.1.3052.12.1.1.1.1.6.
+                    descr: esPointName
+                    skip_values:
```

Comment:
Hmm, I thought about having the format like this:
`{{ $esIndexPC }} != 1`  and parsing it out, but I'm not sure which is better.

Reply:
Yeah I saw your comment in the other PR but that relies on good formatting and a good parser. I chose to make things specific which should lead to less issues with a negligible increase in yaml size.